### DISCUSSION TITLE
Why "use client" on the main page?

### DISCUSSION THREAD

**Discussion by labintsev** [scaffold-eth/scaffold-eth-2#1018] [Q&A] [answered]
Why do we "use client" on the main page?
 I think for SEO and performance reasons it may be SSR. 
Can we move useAccount()  wagmi hook to the Address component?

**Comment by technophile-04** [comment, marked as answer]
Thanks @labintsev, actually we wanted to have basic example of Address component on homepage without making the home page itself complicated for tweak (like moving client component in other dir) for beginner
